models/iaf_psc_alpha_multisynapse.h

```diff
@@ -158,6 +162,27 @@ class iaf_psc_alpha_multisynapse : public Archiving_Node
    */
   struct State_
   {
+
+    /**
+     * Enumeration identifying elements in state vector State_::y_.
+     * This enum identifies the elements of the vector. The last element
+     * of this (I_SYN, ) will be repeated n times at the end of the
+     * state vector State_::y with n being the number of synapses.
+     * As a first step this vector will only be used for logging.
+     */
+    enum StateVecElems
+    {
+      V_M = 0,
+      I,     // 1
+      I_SYN, // 2
+      STATE_VECTOR_MIN_SIZE
+    };
+
+    static const size_t NUMBER_OF_FIXED_STATES_ELEMENTS = 2; // V_M, I
+    static const size_t NUM_STATE_ELEMENTS_PER_RECEPTOR = 1; // I_SYN
+
+    std::vector< double > y_; //!< neuron state
```

Comment:
Introducing this vector only for the sake of recording seems not a good idea, as it costs both memory and runtime. I would suggest to make the accession functor smarter, along the lines of 
```
if ( elem == V_M )
{ return S_.V_m_ }
else 
{ return S_.y2_syn_[elem/2]; }
```
NB: I am not quite sure if my index arithmetic above is correct, this is just meant as a suggestion.